doc/ss_logger.rst

```diff
@@ -0,0 +1,144 @@
+****************
+SmartSim Logging
+****************
+========
+Overview
+========
+SmartSim supports experiment tracking with logging functionality
+using the SmartSim log module. The logger, supported by Python logging, enables
+monitoring the experiment at runtime by allowing users to print messages **to stdout**
+and/or **to file** from within the Python driver script. The SmartSim logger allows users to categorize
+messages by severity level. Users may instruct the SmartSim to print
+specified severity levels and ignore others through the `SMARTSIM_LOG_LEVEL`
+environment variable.
+
+The `SMARTSIM_LOG_LEVEL` environment variable accepts **three** inputs,
+ranked from lowest severity (1) to highest (3). Each severity level has
+associated logging functions users may use within an experiment driver script:
```

Comment:
I think the description (or bullet points) might be reversed. bullet 1 is described as "lowest severity" but covers errors and warnings. I'd suggest reversing content in bullets since low->high is common

Nit: the focus on (1) to (3) and numeric bullets seem to imply more than just ordering. I might be overthinking, but it feels like we're almost suggesting the underlying integer value associated with the python log levels. How would you feel about doing away with the specific parts referencing numbers "(1) to ... (3)" in the description and just use bullets instead of numbers in the list?